examples/finnegans_wake_demo/finnegans-wake-demo-lynx.py

```diff
@@ -0,0 +1,191 @@
+"""
+ This file is part of nucypher.
+
+ nucypher is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Affero General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ nucypher is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
+"""
+
+import datetime
+import maya
+import os
+from umbral.keys import UmbralPublicKey
+from web3.main import Web3
+
+from nucypher.blockchain.eth.signers.base import Signer
+from nucypher.characters.lawful import Alice, Bob, Ursula
+from nucypher.characters.lawful import Enrico as Enrico
+from nucypher.crypto.powers import SigningPower, DecryptingPower
+from nucypher.utilities.logging import GlobalLoggerSettings
+
+# Twisted Logger
+GlobalLoggerSettings.set_log_level(log_level_name='debug')
+GlobalLoggerSettings.start_console_logging()
+
+# Execute the download script (download_finnegans_wake.sh) to retrieve the book
+BOOK_PATH = os.path.join('.', 'finnegans-wake.txt')
+
+# Change this value to to perform more or less total re-encryptions
+# in order to avoid processing the entire book's text. (it's long)
+NUMBER_OF_LINES_TO_REENCRYPT = 25
+
+# Seednode
+SEEDNODE_URI = "https://lynx.nucypher.network:9151"
```

Comment:
Can we fetch this from `RestMiddleware.TEACHER_NODES`?

Reply:
Yes, but I think there's a small tradeoff for readability there. IMO default seed URLs need to move somewhere else especially considering that they are entirely unused in `RestMiddleware`.

Reply:
:heavy_check_mark: - used `Ursula.seednode_for_network` instead.